index.html

```diff
@@ -165,7 +165,19 @@ <h4 class="title">Skills</h4>
 					<img src="images/Union2.svg" alt=".">
 			</li>
 		</ul>
+	</div>
 
+	<div class="contact">
+		<section class="contact-header">
+			<h2>Contact me</h2>
+			<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+		</section>
+		<form action="#" class="contact-form">
+			<input type="text" placeholder="Your name" required>
+			<input type="email" name="email" id="email" placeholder="Your email" required>
+			<textarea name="text" id="text" cols="30" rows="10" placeholder="Write your message here" required></textarea>
+			<button type="submit" class="button btn">Get In Touch</button>
+		</form>
```

Comment:
- [ ] You did a great job on the design but there are a few issues in the form you need to fix.
  - Please make sure the name field only accepts up to 30 chars. you can use `maxlength` property for that. hint: `maxlength="30".
  - Please make sure the message field only accepts up to 500 chars. you can use `maxlength` property for that. hint: `maxlength="500".
  - Please collect the user data using [Formspree service](https://formspree.io/html). You should create an account there and place the formspree ID in the action attribute and use `post` as a method. hint: ` <form action="https://formspree.io/f/YOURID" method="post">`